.github/workflows/update-links.yml

```diff
@@ -31,15 +32,17 @@ jobs:
         env:
           CF_ACCESS_KEY_ID: ${{ secrets.CF_ACCESS_KEY_ID }}
           CF_SECRET_ACCESS_KEY: ${{ secrets.CF_SECRET_ACCESS_KEY }}
-
-      - name: Open PR with updates
-        uses: peter-evans/create-pull-request@v5
-        with:
-          commit-message: Update Redirect links
-          title: "chore: update redirect links"
-          body: |
-            This PR updates the links for url redirects.
-          branch: update-redirect-links
-          delete-branch: true
-          base: main
+      
+      - name: Commit Changes
+        # Not sure if we need to define user.name and user.email in git config
+        run: |
+          git add src/constants/redirectLinks.json
+          git commit -m 'chore: update redirect links'
+          git push
```

Comment:
This will not work. I recommend using an action such as https://github.com/nodejs/nodejs.org/blob/main/.github/workflows/translations-pr.yml#L104-L108